cargo-shuttle/src/init.rs

```diff
@@ -425,7 +425,10 @@ type GetDependencyVersionFn = fn(&str, &Path, &Url) -> String;
 /// Gets the latest version for a dependency of `crate_name`.
 /// This is a wrapper function for `cargo_edit::get_latest_dependency` function.
 fn get_latest_dependency_version(crate_name: &str, manifest_path: &Path, url: &Url) -> String {
-    let latest_version = get_latest_dependency(crate_name, false, manifest_path, Some(url))
+    // some crates need to be on the prerelease version
+    let flag_allow_prerelease = matches!(crate_name, "rocket");
```

Comment:
This does fix it for rocket, but I'm not sure if this is the best approach for long-term maintenance though. This function will be "injected" into the template generation process. It, therefore, doing this "side-effect" seem to be a bit unexpected.

Can't a `bool` rather be added to the argument list of `GetDependencyVersionFn`? This will mean also updating `set_key_value_dependency_version`

Reply:
You are right, doing it within the template makes more sense. I did consider that, but then figured that we already had the `crate_name` in the arguments. 

Maintainance-wise is simpler to just add the boolean in the template, yes.

Will update soon